src/bootstrap/src/lib.rs

```diff
@@ -489,7 +489,7 @@ impl Build {
 
             // Make sure we update these before gathering metadata so we don't get an error about missing
             // Cargo.toml files.
-            let rust_submodules = ["library/backtrace", "library/stdarch"];
+            let rust_submodules = ["ferrocene/library/backtrace-rs", "library/stdarch"];
```

Comment:
We should not add `ferrocene/library/backtrace-rs` to the list of submodules to update, since it's not a submodule anymore.